README.md

```diff
@@ -126,3 +126,13 @@ It should be stated that the abstraction layer (currently `NAN`) should
 support all *current* LTS releases. Given that Active LTS will overlap
 for a period of 6 months, this means that the abstraction layer will, at
 any given point in time, fully support a maximum of 2 LTS releases.
+
+## LTS Team members
+
```

Comment:
If the purpose is to provide contact information for people outside of the org, then linking to a page that only org people can see isn't very useful (people in the org, I would assume, already know how to look up the LTS org team in GitHub)